integration/kubernetes/busybox.bats

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	busybox_image="busybox"
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	first_pod_name="first-test"
+	second_pod_name="second-test"
+	sleep_cmd="sleep 30"
+	# Pull the images before launching workload. This is mainly because we use
+	# a timeout and in slow networks it may result in not been able to pull the image
+	# successfully.
+	sudo -E crictl pull "$busybox_image"
+}
+
+@test "Check UTS and IPC namespaces" {
+	wait_time=30
+	sleep_time=5
+	# Run the first pod
```

Comment:
I find this very hard to read as it's all in a single block - could you maybe add some blank lines to break the block into logical chunks/paragraphs?